### ISSUE TITLE
I think you have a error.

### ISSUE THREAD

**Issue by typhoon1104** [SSARCandy/DeepCORAL#19]
DEEP_CORAL_LOSS: 
def CORAL(source, target):
    d = source.data.shape[1]
    ns = source.data.shape[0]
    nt = target.data.shape[0]

    # source covariance
    xm = torch.mean(source, 0, keepdim=True) - source
    xc = (xm.t() @ xm) / (ns-1)

    # target covariance
    xmt = torch.mean(target, 0, keepdim=True) - target
    xct = xmt.t() @ xmt / (nt-1)

    print(xc, xct)

    # frobenius norm between source and target
    loss = torch.sum(torch.mul((xc - xct), (xc - xct)))
    loss = loss/(4*d*d)
    return loss

**Comment by ZhouWenjun2019** [comment]
I don't understand this part, too. Do you have some idea?

**Comment by ch-andrei** [comment]
tldr, no error, this code is "correct". See my answer for this [issue ](https://github.com/SSARCandy/DeepCORAL/issues/17).
